packages/shared/topology/factory/StyleNode.tsx

```diff
@@ -271,6 +271,8 @@ const StyleNode: React.FunctionComponent<StyleNodeProps> = ({
             detailsLevel === ScaleDetailsLevel.high &&
             passedData.showStatusDecorator
           }
+          // eslint-disable-next-line @typescript-eslint/no-empty-function
+          onStatusDecoratorClick={() => {}}
```

Comment:
Just an observation: you can avoid disabling the linter by returning null
```suggestion
          onStatusDecoratorClick={() => null}
```